### PR TITLE
refactor: preview for components

### PR DESCRIPTION
**Pull request by Oshioke-Salaki**
Refactor preview for leaderboard, add token and hamburger components

### CODE REVIEW

File: app/utilities/Iframe.tsx [hunk 1]
```diff
@@ -64,6 +64,17 @@ const Iframe = ({ children, ...props }: { children: ReactNode }) => {
         style.appendChild(iframeDoc.createTextNode(customStyles));
         iframeDoc.head.appendChild(style);
 
+        iframeDoc.body.classList.add(
```
Comment (stephanniegb) [inline app/utilities/Iframe.tsx:67]: Why did you add this?

Comment (Oshioke-Salaki) [reply app/utilities/Iframe.tsx:67]: To center the the component in the preview body.

**Review by stephanniegb — APPROVED** [review]
